.github/workflows/ci.yml

```
@@ -30,3 +30,29 @@ jobs:
   package:
     name: Python package
     uses: beeware/.github/.github/workflows/python-package-create.yml@main
+
+  test:
+    name: Test Python package
+    needs: package
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Get packages
+        uses: actions/download-artifact@v4.1.4
+        with:
+          name: ${{ needs.package.outputs.artifact-name }}
+          path: dist
+
+      - uses: actions/setup-python@v5
```

We use explicit version pins to ensure there's reproducibility in CI environments.
```suggestion
      - uses: actions/setup-python@v5.1.0
```